### PR TITLE
Ft sign in to phantom

### PR DESCRIPTION
**Pull request by tjeaneric**
#### What does this PR do?
* Sign in to phantom
* Administrator to use pre-configured login credentials

#### Description of Task to be completed?
* Sign in to phantom where the Administrator will use pre-configured login credentials.
* Have the following endpoint working
`POST /api/v1/user/login`

#### How should this be manually tested?
* `git pull` this branch 
* run `npm install`
* check env.example file and add JWT_SECRET and JWT_EXPIRES_IN to your .env file
* Do migrations with `npm run migrate`
* run `npx sequelize-cli db:seed:all` to create pre-configured credentials for seeders
* run `npm run dev`
* Test the route `POST /api/v1/user/login` in POSTMAN or any other platform for using API

#### Any background context you want to provide?
1. I used the JWT package for hashing the password stored in the Database
2. Documentation can be accessed at `/api-docs`

#### What are the relevant pivotal tracker stories?
*  **The driver, administrator, and operator** should be able to sign in to the app by providing their email & password.
* The API should return a JW token on a successful login.
* The **administrator** should be able to sign in to the app using pre-configured login credentials.
* Appropriate error messages are displayed on unsuccessful attempts

#### Questions:
No.

### CODE REVIEW

**Review by kwizeradev — APPROVED** [review]

File: src/database/seeders/20220418183506-demo-user.js [hunk 1]
```diff
@@ -1,34 +1,32 @@
-'use strict';
+import bcrypt from 'bcryptjs';
+
+// const hashPassword = async (plainPassword) => {
+//   const hash = await bcrypt.hash(plainPassword, 12);
+//   return hash;
+// };
```
Comment (Angelus123) [inline src/database/seeders/20220418183506-demo-user.js]: Remove unnecessary!

**Review by Angelus123 — APPROVED** [review]

**Review by furebo — CHANGES_REQUESTED** [review]
resolve the conflicts

File: src/database/seeders/20220418183506-demo-user.js (hunk 1 above)
Comment (tjeaneric) [reply src/database/seeders/20220418183506-demo-user.js]: Done.

**Comment by tjeaneric** [comment]
> 

Resolved

**Review by gerard1001 — APPROVED** [review]

**Review by Kalimbak — APPROVED** [review]

**Review by furebo — APPROVED** [review]

**Review by furebo — APPROVED** [review]

**Pull request merged**